unittest/unit/exporter.js

```diff
@@ -59,16 +60,8 @@ const REPORTS = [
   }
 ];
 
-describe('GitHub export', () => {
+describe('GitHub export', async () => {
```

Comment:
```suggestion
describe('GitHub export', () => {
```

Last I tried, async functions for `describe` doesn't do what one would hope, the framework doesn't wait for the promise to resolve, so it really does nothing. Hopefully not needed?

Although... it is used below. Maybe that's actually broken too?

Reply:
This was probably a leftover from when I had been trying to configure the `proxyquire` alternative.